tests/integration/commands/proxy.cairo

```diff
@@ -0,0 +1,128 @@
+%lang starknet
```

Comment:
The original test accroding to its name has been also testing on proxy, don't we have this file somewhere already?

Reply:
We had compiled one (json). I think having source is better, had to debug this one and really couldn't trace back which implementation it was (to re-compile) so i put in the source.

Reply:
Agreed!